src/clusterfuzz/_internal/bot/tasks/setup.py

```diff
@@ -223,7 +223,7 @@ def setup_testcase(testcase,
       error_message = 'Fuzzer %s no longer exists' % fuzzer_name
       data_handler.update_testcase_comment(testcase, data_types.TaskState.ERROR,
                                            error_message)
-      return None, None, None
+      return None, None, uworker_errors.Error(uworker_errors.Type.NO_FUZZER)
```

Comment:
Can the docstring for this function have a line describing what the three return values are?  As someone reading the code, it's hard to tell what the "None", "None" parts stand for & if they'll always be None & what the purpose there is

Reply:
+1